lib/bolt/inventory/inventory.rb

```diff
@@ -140,7 +146,8 @@ def expand_targets(targets)
           targets.map { |tish| expand_targets(tish) }
         when String
           # Expand a comma-separated list
-          targets.split(/[[:space:],]+/).reject(&:empty?).map do |name|
+          # Regex magic below is required to workaround `{foo,bar}` glob syntax
+          targets.scan(/[^\s,]*(?:(?<={).*?(?=}))[^\s,]*|[^\s,]*/).reject(&:empty?).map do |name|
```

Comment:
Let's use a named constant (e.g. `TARGET_REGEX` or similar) and use that here instead of inline regex.